### PR TITLE
Clone or move some Turkish shops from supermarket file to convenience

### PR DESCRIPTION
**Pull request by evgenykatyshev**

### CODE REVIEW

File: data/brands/shop/convenience.json [hunk 1]
```diff
@@ -3170,6 +3192,28 @@
         "shop": "convenience"
       }
     },
+    {
+      "displayName": "Migros",
+      "id": "migros-986a24",
+      "locationSet": {"include": ["001"]},
```
Comment (kjonosm) [inline data/brands/shop/convenience.json:3198]: `locationSet` should be limited to `tr`

File: data/brands/shop/convenience.json [hunk 2]
```diff
@@ -3170,6 +3192,28 @@
...
+      "tags": {
+        "brand": "Migros",
+        "brand:wikidata": "Q680727",
```
Comment (kjonosm) [inline data/brands/shop/convenience.json:3201]: `brand:wikidata` should also refer to `Q1754510`

**Review by kjonosm — CHANGES_REQUESTED** [review]

File: data/brands/shop/convenience.json (hunk 1 above)
Comment (evgenykatyshev) [reply data/brands/shop/convenience.json]: Done

File: data/brands/shop/convenience.json (hunk 2 above)
Comment (evgenykatyshev) [reply data/brands/shop/convenience.json]: Changed wikidata reference

**Review by kjonosm — APPROVED** [review]

**Comment by kjonosm** [comment]
Thanks @evgenykatyshev

**Pull request merged**